### PR TITLE
[CLI] Add user revoke command.

### LINKED ISSUE #9143: [CLI] expose user revocation

### PR DESCRIPTION
**Pull request by AureliaDolo**
Close #9143

### CODE REVIEW

File: cli/src/commands/user/revoke.rs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    let to_revoke = users
+        .iter()
+        .find(|info| info.human_handle.email() == email)
+        .ok_or(anyhow!("User not found"))?;
+    client.revoke_user(to_revoke.id).await?;
+
+    println!("User {email} has been revoked");
+
+    client.stop().await;
+
```
Comment (FirelightFlagboy) [inline cli/src/commands/user/revoke.rs:30]: Handled by `build_main_with_client`

**Comment by FirelightFlagboy** [comment]
Missing newsfragment

**Review by vxgmichel — APPROVED** [review]
I agree with @FirelightFlagboy , LGTM otherwise :+1:

File: cli/src/commands/user/revoke.rs (hunk 1 above)
Comment (AureliaDolo) [reply cli/src/commands/user/revoke.rs:30]: It has been forgotten in quite a few places #9163

File: newsfragments/9143.feature.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+[CLI] Add user revoke command.
```
Comment (FirelightFlagboy) [inline newsfragments/9143.feature.rst:1]: **nitpick**: I write it that way

```suggestion
Add CLI command ``user revoke`` to revoke a user
```

File: cli/src/commands/user/revoke.rs [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+use anyhow::anyhow;
+
+use crate::utils::*;
+
+crate::clap_parser_with_shared_opts_builder!(
+    #[with = config_dir, device, password_stdin]
+    pub struct Args {
+        #[arg(long)]
+        /// Email of the user to revoke
+        email: String,
```
Comment (FirelightFlagboy) [inline cli/src/commands/user/revoke.rs:12]: **nitpick**: For consistency, the doc-comment is before fields attributes

**Review by FirelightFlagboy — APPROVED** [review]

**Review by FirelightFlagboy — APPROVED** [review]

**Pull request merged**